modules/layout/src/lib/header/header.component.html

```diff
@@ -11,29 +15,36 @@
         id="logo"
       />
     </a>
-    <button
-      id="menu-button"
-      (click)="toggleMenu()"
-      title="Botão abrir/fechar menu"
-    >
-      <svg
-        width="26"
-        height="23"
-        viewBox="0 0 26 23"
-        fill="none"
-        xmlns="http://www.w3.org/2000/svg"
+    <div id="menu-toggle-container">
+      <h2 id="menu-title" class="mr-1rem">Menu Principal</h2>
```

Comment:
```suggestion
      <h2 id="menu-title" class="mr-1rem">Menu</h2>
```
Como só temos um Menu, acho que podemos omitir o texto "Principal" dele 😉

Reply:
Título atualizado e menu corrigido:
![image](https://github.com/user-attachments/assets/0806400e-d8a8-4d4a-80a0-2dfeecfdff2a)